volatility3/framework/symbols/linux/__init__.py

```diff
@@ -307,15 +307,23 @@ def files_descriptors_for_process(
         symbol_table: str,
         task: interfaces.objects.ObjectInterface,
     ):
-        # task.files can be null
-        if not (task.files and task.files.is_readable()):
+        try:
```

Comment:
This doesn't need the exception handler setting up twice, all the calls can happen with the try/except and any failure within that block will result in a return `None`.